### DISCUSSION TITLE
Authentication interceptor no longer works after nginx upgrade 1.8.1 -> 1.9.0

### DISCUSSION THREAD

**Discussion by anurag-appperfect** [nginx/kubernetes-ingress#6041] [Q&A]
After upgradation of ingress-controller from 1.8.1 -> 1.9.0, interceptor is not working for some of the API's. We verified that nginx is successfully deployed in the kubernetes cluster. We have done some minor changes to nginx.tmpl configuration file:
HttpAccessLogPath: variable renamed to HTTPAccessLogPath
location.Whitelist: variable renamed to location.Allowlist and location.Denylist
EnableRealIp: variable renamed to EnableRealIPafter deployment

Community please help us to get this issue resolve

**Comment by pdabelf5** [comment]
This appears to be related to the [community ingress](https://github.com/kubernetes/ingress-nginx) project.
